miden/tests/integration/operations/u32_ops/arithmetic_ops.rs

```diff
@@ -754,8 +745,8 @@ proptest! {
     }
 
     #[test]
-    fn u32addc_proptest(a in any::<u32>(), b in any::<u32>(), c in 0_u32..1) {
-        let asm_op = "u32addc";
+    fn u32add3_proptest(a in any::<u32>(), b in any::<u32>(), c in 0_u32..1) {
```

Comment:
we should change for `c` to also be an arbitrary `u32` value.

Reply:
Yes, I missed this. Thanks. 

Please see [0c2cb4f](https://github.com/maticnetwork/miden/pull/221/commits/0c2cb4faceaf1058b902328f5cb2889639ff1943).